tests/test_dockerflow.py

```diff
@@ -23,9 +23,8 @@ def test_dockerflow_version_matches_disk_contents(client, versionfile):
 
 
 def test_heartbeat_returns_200(
-    client, db, phabdouble, request_mocker, redis_cache, jwks, treestatusdouble
```

Comment:
`db` here was causing a strange app context error during teardown. Going to dig a little as to whether this is going to cause any real problems or whether we should return the db fixture here, but the error was happening on teardown.